packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js

```diff
@@ -1,6 +1,62 @@
 export const COLD_ARCHIVE_TRACKING_PREFIX =
   'PublicCloud::pci::projects::project::storages::cold_archive';
 
+export const COLD_ARCHIVE_TRACKING = {
+  PREFIX: 'PublicCloud::pci::projects::project::storages::cold_archive',
+  ONBOARDING: {
+    MAIN: 'onboarding',
+    ADD_CONTAINER: 'add',
+    DOCUMENTATION_LINK: 'docs',
+  },
+  ADD: {
+    MAIN: 'add',
+    PREFIX: 'PublicCloud_cold_archive_add_archive::create_container',
+    ASSOCIATE: {
+      NEW_USER: 'add_new_user',
+      EXISTING_USER: 'add_existing_user',
+      ASSOCIATE_USER: 'associate_user',
+    },
+    CREATE: 'create_container',
+  },
+  CONTAINERS: {
+    MAIN: 'dashboard::containers',
+    RESTORE: 'restore',
+    DELETE: 'delete',
+    ADD_USER: 'add-user',
+    ARCHIVE: 'archive',
+    DELETE_CONTAINER: 'delete-container',
```

Comment:
delete-container -> delete_container

Reply:
Following the specification, it's "delete-container".